examples/ex02_cksum/test/ChecksumCL_test.py

```diff
@@ -187,18 +189,8 @@ def setup_class( cls ):
   # runs test. We can overwrite this function when inheriting from the
   # test class to apply different passes to the DUT.
 
-  def run_sim( s, th, max_cycles=1000 ):
-
-    # Create a simulator
-    th.apply( DefaultPassGroup() )
-    th.sim_reset()
-
-    # Tick the simulator
-    while not th.done() and th.sim_cycle_count() < max_cycles:
-      th.sim_tick()
-
-    # Check timeout
-    assert th.sim_cycle_count() < max_cycles
+  def run_sim( s, th ):
+    run_sim( th, s.__class__.cmdline_opts )
```

Comment:
Do you think we should tell people to use `run_sim` whenever possible?

Reply:
I think so?